src/ocaml/utils/profile.ml

```diff
@@ -147,6 +147,16 @@ let record ?accumulate pass f x = record_call ?accumulate pass (fun () -> f x)
 let record_with_counters ?accumulate ~counter_f pass f x =
   record_call_internal ?accumulate ~counter_f pass (fun () -> f x)
 
+let file_prefix = "file="
+(*
+let annotate_file_name name =
+  let file_path =
+    match !Clflags.directory with
+    | Some directory -> Filename.concat directory name
+    | None -> name
+  in
+  file_prefix ^ file_path
+*)
```

Comment:
Nit: it would be slightly friendlier to ourselves to explain why we commented this out.

Reply:
(pushed a change to explain this)